courses/free-courses-en.md

```diff
@@ -3,7 +3,8 @@
 * [Algorithms & Data Structures](#algorithms--data-structures)
 * [Android](#android)
 * [Angular](#angular)
-* [Assembly](#assembly)
+* [Assembly Language](#assembly-language)
+  * [Non-X86](#non-x86) 
```

Comment:
In general we try to avoid single item listing.

Could be moved to top "Asembly"